webrick/upstart/tasks/RedHat.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: systemd unit - RedHat
+  template:
+    dest: "{{ webrick_systemd_unit }}"
+    src: systemd.service.j2
```

Comment:
Right now userjobs has the following benefits:
1. The "deploy"-user can start, stop and restart the job
2. The "deploy"-user can change the job (eg. add or change environment variables)
3. The user job is started during the boot process.

Most required it the restart option.

So far I could not figure out a good way to mirror this behavior.